python/cuml/experimental/explainer/common.py

```diff
@@ -146,13 +149,15 @@ def get_link_fn_from_str_or_fn(link):
     return link_fn
 
 
-# temporary function while explainers adopt decorators and cumlarray descriptor
 def output_list_shap_values(X, dimensions, output_type):
     if output_type == 'cupy':
         if dimensions == 1:
             return X[0]
         else:
-            return X
+            res = []
```

Comment:
Super picky but this seems like either a list comprehension or just `list(X)` would be nicer